lib/clock.js

```diff
@@ -1,19 +1,23 @@
+var util = require('./util');
+
 function SdkClock(localOffset) {
   // Calculated local clock offset from server time (in milliseconds). Can be positive or negative.
   this.localOffset = parseInt(localOffset || 0);
 }
 
-Object.assign(SdkClock.prototype, {
+util.extend(SdkClock.prototype, {
```

Comment:
How is util.extend different than Object.assign()?

Reply:
It's our own naive property cloner

Reply:
ah, IE11.  :shakefist: